### PR TITLE
Support videomatte dataset for background_matting models

### PR DESCRIPTION
**Pull request by akorobeinikov**
Support datasets with separately `pha` and 3 channel `fgr`

### CODE REVIEW

File: tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/background_matting.py [hunk 1]
```diff
@@ -144,6 +144,10 @@ def parameters(cls):
                 'background_prefix': StringField(optional=True, default='', description='prefix for gt backgrounds'),
                 'background_postfix': StringField(optional=True, default='.png', description='prefix for backgrounds'),
                 'with_background': BoolField(optional=True, default=False, description='load backgrounds'),
+                'foregrounds_dir': PathField(optional=True, description='path to input foregrounds directory', is_directory=True),
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/background_matting.py:147]: please update converter readme

**Review by eaidova — APPROVED** [review]

**Comment by eaidova** [comment]
@akorobeinikov please fix code style

File: tools/accuracy_checker/openvino/tools/accuracy_checker/representation/segmentation_representation.py [hunk 2]
```diff
@@ -376,20 +376,47 @@ class SalientRegionPrediction(SegmentationPrediction):
...
+            data_source = self.metadata.get('additional_data_source')
+            if data_source is None:
+                data_source = self.metadata['data_source']
+            if isinstance(loader_config, str):
+                loader = BaseReader.provide(loader_config, data_source)
+            else:
+                loader = BaseReader.provide(loader_config['type'], data_source, config=loader_config)
+            fgr = loader.read(self._foreground_path)
+            return fgr.astype(np.uint8)
+        elif self._fgr is None:
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/representation/segmentation_representation.py:415]:
```suggestion
        if self._fgr is None:
```

File: tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/background_matting.py [hunk 3]
```diff
@@ -159,7 +161,7 @@ def update(self, annotation, prediction):
         if pred.shape[-1] == 1 and pred.shape[-1] != gt.shape[-1]:
             gt = cv2.cvtColor(gt, cv2.COLOR_RGB2GRAY)
         if self.use_mask:
-            mask = self.prepare_pha(annotation.value) > 0
+            mask = self.prepare_pha(annotation) > 0.01
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/background_matting.py:164]: can we have some parameter for changing tolerance?

Comment (akorobeinikov) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/background_matting.py]: added new parameter

**Pull request merged**